internal/resource-request-operator/broadcaster.go

```diff
@@ -314,7 +422,32 @@ func updateResources(currentResources, oldResources, newResources corev1.Resourc
 	}
 }
 
-func extractPodResources(pod *corev1.Pod) corev1.ResourceList {
-	resourcesToExtract, _ := resourcehelper.PodRequestsAndLimits(pod)
+func extractPodResources(podToExtract *corev1.Pod) corev1.ResourceList {
+	resourcesToExtract, _ := resourcehelper.PodRequestsAndLimits(podToExtract)
```

Comment:
Would personally revert this change.

Reply:
use pod as variable name shadows the import from pod package.